### PR TITLE
Bubble up to pkg/volume/OWNERS file

### PR DESCRIPTION
**Pull request by xing-yang**
#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:
This PR removes OWNERS file under pkg/volume/csi and pkg/volume/testing and bubbles up to pkg/volume/OWNERS.

#### Which issue(s) this PR fixes:
Fixes #

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:
```docs

```

### CODE REVIEW

File: pkg/volume/csi/OWNERS [hunk 1]
```diff
@@ -1,7 +1,11 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-- jsafrane
-- saad-ali
-- vladimirvivien
-- davidz627
+- sig-storage-approvers
```
Comment (msau42) [inline pkg/volume/csi/OWNERS:4]: Should we just delete this file and let it bubble up to pkg/volume/OWNERS?

Comment (xing-yang) [reply pkg/volume/csi/OWNERS:4]: Sure.

**Pull request merged**